pkgs/top-level/aliases.nix

```diff
@@ -1509,7 +1510,7 @@ mapAliases ({
   xmonad_log_applet_gnome3 = throw "'xmonad_log_applet_gnome3' has been renamed to/replaced by 'xmonad_log_applet'"; # Converted to throw 2022-02-22
   xmpp-client = throw "xmpp-client has been dropped due to the lack of maintanence from upstream since 2017"; # Added 2022-06-02
   xmpppy = throw "xmpppy has been removed from nixpkgs as it is unmaintained and python2-only";
-  xp-pen-g430 = pentablet-driver; # Added 2020-05-03
+  xp-pen-g430 = xp-pen-g430-driver; # Changed to xp-pen-g430-driver 2022-06-23
```

Comment:
The comment shouldn't be updated here. Seeing how the entry was made in 2020, i also think it can be moved to a `throw`.